tests/test_room_resource/test_delete_resource.py

```diff
@@ -2,33 +2,57 @@
 import os
 
 from tests.base import BaseTestCase, CommonTestCases
-from fixtures.room_resource.delete_room_resource import (  # noqa: F401
-  delete_resource, expected_query_after_delete, delete_non_existant_resource)  # noqa: E501
-from helpers.database import db_session  # noqa: F401
+from fixtures.room_resource.delete_room_resource import (
+  delete_resource, expected_response_after_delete, delete_non_existent_resource)
 
 
 sys.path.append(os.getcwd())
 
 
 class TestDeleteRoomResource(BaseTestCase):
 
-    def test_deleteresource_mutation_when_not_admin(self):
+    def test_delete_resource_mutation_when_not_admin(self):
+        """
+             Test delete resource when one is not an admin
+        """
         CommonTestCases.user_token_assert_in(
             self,
             delete_resource,
             "You are not authorized to perform this action"
         )
 
     def test_delete_resource_mutation_when_admin(self):
-        CommonTestCases.admin_token_assert_in(
+        """
+            Test successful delete of a resource by an admin
+        """
+        CommonTestCases.admin_token_assert_equal(
             self,
             delete_resource,
-            "Markers"
+            expected_response_after_delete
+        )
+
+    def test_delete_non_existent_resource(self):
+        """
```

Comment:
Are you testing whether it is possible to delete or not? Please describe what is being tested

Reply:
This has been worked on @Bonifase